packages/http-server/src/getHttpConfigFromRequest.ts

```diff
@@ -6,11 +6,15 @@ import * as D from 'io-ts/lib/Decoder';
 import * as parsePreferHeader from 'parse-prefer-header';
 
 const BooleanFromString = D.parse<string, boolean>(s =>
-  s === 'true' ? D.success(true) : s === 'false' ? D.success(false) : D.failure(s, 'Not a boolean')
+  s === 'true' ? D.success(true) : s === 'false' ? D.success(false) : D.failure(s, 'a boolean')
```

Comment:
The error message when this failed was saying `should be Not a boolean`, so updated so that the error message would read `should be a boolean`.